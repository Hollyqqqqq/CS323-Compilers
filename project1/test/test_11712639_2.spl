int test(){
    int a = 10ï¼›
    int 00ad = 2;
    int 0a = 2;
    int b = 0x0a;
    int c = 0xg;
    char d = '@';
    char e = 'a;
    char f = '@@';
    char g = '\x04';
    float h = 0.0;
    return 0abc@dfa;
}