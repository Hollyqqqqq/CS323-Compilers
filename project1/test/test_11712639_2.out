Error type A at Line 2: Chinese SEMI
Error type A at Line 3: Wrong id 00ad
Error type A at Line 4: Wrong id 0a
Error type A at Line 5: wrong hex int 0x0a
Error type A at Line 6: wrong hex int 0xg
Error type A at Line 8: unknown lexeme 'a
Error type A at Line 9: wrong char '@@'
Error type A at Line 10: wrong char '\x04'
Error type A at Line 12: unknown lexeme 0abc@dfa
